### PR TITLE
ON1_DotProductandAnglebetweentwovector

### PR DESCRIPTION
**Pull request by mlamich1**

### CODE REVIEW

File: linear-algebra/source/future-ON/01.ptx [hunk 1]
```diff
@@ -148,8 +148,32 @@ The dot product combines two vectors and creates a scalar that gives us geometri
...
-    
-    
+        <definition>
+            <statement>
+                <p> Given two vectors <m>\vec u</m> and <m>\vec v</m> in <m>\IR^n</m>, such that  <m>\vec u</m> and <m>\vec v</m>  are not parallel, let <m>\theta</m> be the  <term>angle</term> between the two vectors, 
+                    then 
+                    <me>
+                        \cos \theta = \frac{\vec u \cdot \vec v }{|\vec u||\vec v|}.
+                    </me>
+                     By Law of Cosine:
```
Comment (jbunn3) [inline linear-algebra/source/future-ON/01.ptx:158]: Maybe say, By the Law of Cosines

File: linear-algebra/source/future-ON/01.ptx [hunk 2]
```diff
@@ -148,8 +148,32 @@ The dot product combines two vectors and creates a scalar that gives us geometri
...
+                    Using dot product of vector  <m>\vec u-\vec v</m> with itself:
+                  
+                    <md>
+                        <mrow> |\vec u- \vec v|^2 \amp =  (\vec u- \vec v) \cdot (\vec u- \vec v)  \amp </mrow>
+                        <mrow>  \amp =  \vec u \cdot \vec u - 2 ( \vec u \cdot \vec v) + \vec v \cdot \vec v \amp   </mrow>
+                        <mrow>  \amp =  |\vec u|^2 - 2 ( \vec u.\vec v) + |\vec v|^2 \amp   </mrow>
+                      </md> 
+Hence from above, we have:
+<me>
+    \vec u.\vec v = |\vec u||\vec v| \cos \theta .
```
Comment (jbunn3) [inline linear-algebra/source/future-ON/01.ptx:171]: \cdot needed

**Comment by jbunn3** [comment]
@mlamich1 I left a few code comments. You may want to make some changes and do another commit.

**Comment by jbunn3** [comment]
By the way @mlamich1, I don't think your branch was synced up to main before you started these edits. I'm not sure if that will cause a problem.

File: linear-algebra/source/future-ON/01.ptx (hunk 2 above)
Comment (mlamich1) [reply linear-algebra/source/future-ON/01.ptx:171]: Done

File: linear-algebra/source/future-ON/01.ptx (hunk 1 above)
Comment (mlamich1) [reply linear-algebra/source/future-ON/01.ptx:158]: Sure, thanks

**Comment by StevenClontz** [comment]
Merging so that these changes are part of pull request #135

**Pull request merged**